Assets/Scenes/SampleScene.unity

```diff
@@ -339,11 +339,11 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 1cfc7bd3604b9604e951263b773b7934, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  GlobalFont: {fileID: 0}
-  AllPerfectFont: {fileID: 0}
-  SongInfoFont: {fileID: 0}
-  SideInfoFont: {fileID: 0}
-  CenterInfoFont: {fileID: 0}
+  GlobalFont: {fileID: 11400000, guid: 75efbe0ae053d5c4a9a6983514232a25, type: 2}
+  AllPerfectFont: {fileID: 11400000, guid: 75efbe0ae053d5c4a9a6983514232a25, type: 2}
+  SongInfoFont: {fileID: 11400000, guid: 75efbe0ae053d5c4a9a6983514232a25, type: 2}
+  SideInfoFont: {fileID: 11400000, guid: 75efbe0ae053d5c4a9a6983514232a25, type: 2}
+  CenterInfoFont: {fileID: 11400000, guid: 75efbe0ae053d5c4a9a6983514232a25, type: 2}
```

Comment:
There's no need to fill this on, as leaving them *Global only, or Full blank* would make whoever doing building this locally could fit their own values.